src/test/java/uk/gov/hmcts/reform/logging/provider/ErrorCodeJsonProviderTest.java

```diff
@@ -36,4 +36,18 @@ public void testAlertLevel() throws IOException {
         assertThat(node.at("/errorCode").asText()).isEqualTo("0");
         assertThat(node.at("/message").asText()).isEqualTo("test");
     }
+
+    @Test
+    public void testDisableErrorCode() throws IOException, JoranException {
+        System.setProperty("LOGBACK_REQUIRE_ERROR_CODE", "false");
+        captureOutput();
+
+        log.error("no test", new ProviderException("oh no"));
```

Comment:
"no test"?

Reply:
let me update log messages throughout unit tests so it makes sense. not that it matters very much but i can see the point why